openfisca_france/model/prestations/prestations_familiales/paje.py

```diff
@@ -174,9 +174,17 @@ def montant_enfant_ne_apres_avril_2014():
                 )
             return montant
 
+        age_plus_jeune_enfant = min(famille.members('age', period))
+        est_cadet = famille.members('age', period) == age_plus_jeune_enfant
+
+        enfant_elig_avant_avril_2014 = famille.any(enfant_eligible_ne_avant_avril_2014() * est_cadet)
```

Comment:
Merci @frtomas pour ces nouvelles contributions. 
Je suis partisan de garder des noms le moins ambigu possible (et avec le moins d'abbréviations possibles) même s'ils sont parfois très longs.

Reply:
C'est noté @benjello, j'ai changé la variable de la condition pour la rendre plus claire et la faire correspondre avec 'age_plus_jeune_enfant'